src/cargo/ops/registry/search.rs

```diff
@@ -87,5 +90,28 @@ pub fn search(
         );
     }
 
+    if total_crates > 0 {
+        suggest_cargo_info(stdout)?;
+    }
+
+    Ok(())
+}
+
+fn suggest_cargo_info(stdout: &mut dyn Write) -> CargoResult<()> {
+    let literal = LITERAL;
+
+    let _ = writeln!(stdout);
+    note(
+        format_args!("to learn more a package, run `{literal}cargo info <name>{literal:#}`",),
+        stdout,
+    )
+}
+
+pub(super) fn note(msg: impl std::fmt::Display, stdout: &mut dyn Write) -> CargoResult<()> {
```

Comment:
Why is this being used instead of `shell.note`?

Reply:
I didn't notice that. I have updated it.